lib/elixir/lib/calendar/datetime.ex

```diff
@@ -147,16 +147,54 @@ defmodule DateTime do
   If you want the current time in Unix seconds,
   use `System.os_time/1` instead.
 
+  You can also pass a time unit to automatically
+  truncate the resulting datetime. This is available
+  since v1.15.0.
+
   ## Examples
 
       iex> datetime = DateTime.utc_now()
       iex> datetime.time_zone
       "Etc/UTC"
 
+      iex> datetime = DateTime.utc_now(:second)
+      iex> datetime.microsecond
+      {0, 0}
+
+  """
+  @spec utc_now(Calendar.calendar() | :microsecond | :millisecond | :second) :: t
+  def utc_now(calendar_or_time_unit \\ Calendar.ISO) do
+    case calendar_or_time_unit do
+      unit when unit in [:microsecond, :millisecond, :second] ->
+        utc_now() |> truncate(unit)
+
+      calendar ->
+        System.os_time() |> from_unix!(:native, calendar)
+    end
+  end
+
+  @doc """
+  Returns the current datetime in UTC, supporting 
+  a specific calendar and precision.
+
+  If you want the current time in Unix seconds,
+  use `System.os_time/1` instead.
```

Comment:
I don't think we should say this, because you can use `os_time/1` for microsecond/millisecond/nanosecond times as well, and because you can still use `utc_now/1` for that?

Reply:
This is a standard text we have in all `utc_now` functions because people often do `DateTime.utc_now() |> to_unix` :D

Reply:
aaah, gotcha!